.github/workflows/release-please.yml

```diff
@@ -28,12 +37,19 @@ jobs:
           tag=${{ steps.release.outputs.tag_name }}
           printf %s "::set-output name=tag::${tag#v}"
 
-      - name: Build Image
-        if: ${{ steps.release.outputs.release_created }}
```

Comment:
Why did you remove this?

Reply:
Oh, good catch. It should bad copy paste from `build-image.yml`.

Resolved!